### ISSUE TITLE
Rendering mode on Canvas, still tries to load SDF fonts

### ISSUE THREAD

**Issue by promentol** [lightning-js/blits#172]
when setting renderMode to 'canvas', and deleting all sdf fonts, it still tries to load SDF fonts, and throws error

```
Blits.Launch(App, 'app', {
  w: 1920,
  h: 1080,
  multithreaded: false,
  debugLevel: 1,
  reactivityMode: 'Proxy',
  renderMode: 'canvas',
  keymap: keymapping(),
  fonts: [
    { family: 'opensans', type: 'web', file: 'fonts/OpenSans-Medium.ttf' },
  ],
  canvasColor: 'transparent',
  viewportMargin: 100,
})

```

After debugging, inside renderer `fontEngines` is `[class SdfTextRenderer, class CanvasTextRenderer]`

There are many devices which don't support webgl but support canvas, like PS5, or FireTV stick version, canvas rendering is essential.

Thanks

**Comment by promentol** [comment]
@michielvandergeest thank you for commenting

I already made pull request which fix it. While it is a small change, it is easy to reproduce, just render with canvas instead of webgl, and it will throw this error.

https://github.com/lightning-js/blits/pull/173

**Comment by michielvandergeest** [comment]
Released in 1.5.0
